scripts/bundleSize/bundleSizeConfig.js

```diff
@@ -4,6 +4,6 @@ module.exports = {
   // Keep the MAX_SIZE +5 above the largest value and MIN_SIZE -5
   // below the smallest value in the build output; this avoids the
   // need for frequent changes as bundle sizes fluctuate.
-  MIN_SIZE: 587,
-  MAX_SIZE: 871,
+  MIN_SIZE: 593,
+  MAX_SIZE: 894,
```

Comment:
Has this increase been caused by the addition of the library? Seems a lot more than 1KB? I'd say better to avoid the library if it adds this much page weight.

Reply:
Yes, we had to account for the new library. I've revert to doing without `react-router-hash-link`.